app-mobilephone/gammu/gammu-1.38.4.ebuild

```diff
@@ -0,0 +1,60 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils eutils
+
+DESCRIPTION="A tool to handle your cellular phone"
+HOMEPAGE="https://wammu.eu/gammu/"
+IUSE="bluetooth curl dbi debug irda mysql nls odbc postgres usb"
+KEYWORDS="~amd64 ~x86"
+LICENSE="GPL-2"
+
+SLOT="0"
+SRC_URI="https://dl.cihar.com/${PN}/releases/${P}.tar.bz2"
+
+COMMON_DEPEND="
+	dev-libs/glib:2=
+	virtual/libgudev:=
+
+	bluetooth? ( net-wireless/bluez:= )
+	curl? ( net-misc/curl:= )
+	dbi? ( >=dev-db/libdbi-0.8.3:= )
+	mysql? ( virtual/mysql:= )
+	postgres? ( dev-db/postgresql:= )
+	usb? ( virtual/libusb:1= )
+"
+DEPEND="
+	${COMMON_DEPEND}
+	irda?		( virtual/os-headers )
```

Comment:
Please don't align. This is going to look ugly for people with different tabstop.